openfisca_canada/variables/ei/rate_of_benefits/unemployment_benefits__rate_of_weekly_benefits.py

```diff
@@ -0,0 +1,24 @@
+# Import from openfisca-core the common python objects used to code the legislation in OpenFisca
+from openfisca_core.model_api import *
+# Import the entities specifically defined for this tax and benefit system
+from openfisca_canada.entities import Person
+
+class unemployment_benefits__rate_of_weekly_benefits(Variable):
+    value_type = int
+    entity = Person
+    definition_period = Month
+    default_value = .55
```

Comment:
I think there might be a future benefit to using a parameter here, so rather than the default value we can return (for example) parameters(period).rate_of_weekly_benefits and include all the options (assuming this is one of the tables with many options in it)

Reply:
Hello TJ. It make sense. I will get rid of the default value, just return the parameteres instead, thank you!